addon/authenticators/oidc.js

```diff
@@ -106,22 +108,23 @@ export default BaseAuthenticator.extend({
    */
   async _refresh(refresh_token, retryCount = 0) {
     try {
-      const data = await this.get("ajax").post(getUrl(tokenEndpoint), {
-        responseType: "application/json",
-        contentType: "application/x-www-form-urlencoded",
-        data: {
-          refresh_token,
-          client_id: clientId,
-          grant_type: "refresh_token",
-          redirect_uri: this.redirectUri
-        }
+      const body = new URLSearchParams();
+      body.append("refresh_token", refresh_token);
+      body.append("client_id", clientId);
+      body.append("grant_type", "refresh_token");
+      body.append("redirect_uri", this.redirectUri);
+
+      const response = await await fetch(getUrl(tokenEndpoint), {
+        method: "POST",
+        body
```

Comment:
I think we're missing the `headers` containing `accept: "application/json"` and `content-type: "application/x-www-form-urlencoded"` here